t/helper/test-bloom.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * test-bloom.c: test code to exercise bloom.c
+ */
+
+#include "test-tool.h"
+#include "git-compat-util.h"
+#include "bloom.h"
+#include "test-tool.h"
+#include "cache.h"
+#include "commit-graph.h"
+#include "commit.h"
+#include "config.h"
+#include "object-store.h"
+#include "object.h"
+#include "repository.h"
+#include "tree.h"
+
+struct bloom_filter_settings settings = DEFAULT_BLOOM_FILTER_SETTINGS;
+
+static void print_bloom_filter(struct bloom_filter *filter) {
+	int i;
+
+	if (!filter) {
+		printf("No filter.\n");
+		return;
+	}
+	printf("Filter:");
+	printf("\nLength:%d", filter->len);
+	printf("\nData:");
+	for (i = 0; i < filter->len; i++){
+		printf("\n%"PRIu64, filter->data[i]);
```

Comment:
`printf("%08"PRIx64"\n", filter->data[i]);` should give nice fixed-width hex strings here.

Reply:
@derrickstolee  Not really seeing a major benefit by doing this. Maybe I am missing something? Lets take a look tmrw when you are in the office.